pkg/types/types.go

```diff
@@ -24,6 +24,7 @@ type SubmarinerSpecification struct {
 	ColorCodes  []string
 	NatEnabled  bool
 	Broker      string
+	CableDriver string
```

Comment:
I see that we will publish "strongswan" as a "cabledriver" to the broker.

It should really be either "ipsec" or "ipsec/strongswan".


But we can iterate over this on follow up patches.

Reply:
Also, to the broker, what we should probably just share is just Protocol, it doesn't really matter which driver it is as long as it's the same protocol (theoretically, of course..)